ignore_sigpipe.R

```diff
@@ -0,0 +1,10 @@
+library(decor)
```

Comment:
Great to add but, ugh, wish it wasn't needed

Reply:
yeah, total ugh.  we could omit it and run in littler and be fine here. And it will already be fine in the interactive RStudio terminal.  

I could also put it in an .Rprofile or something so that it is run more stealthy, but not sure that's better.  

Longer-term, we can roll this sigpipe handler thing into `neon4cast`.  Because `unify_schemas=FALSE` isn't working yet in arrow, I've opted to bypass `neon4cast::noaa_stage1()` in order to open the parquet directly at the reference_date partition here, which is also kinda ick.